### PR TITLE
test: add frontend e2e test (smoke test for settings page)

### LINKED ISSUE #644: add e2e test for Setting
Setting
- [ ] page jump:
- [ ] set grafana address:

related issue 

resolve #614

### PR DESCRIPTION
**Pull request by idbeta**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] E2E test
- [ ] New feature provided
- [ ] Improve performance

resolve #644

### CODE REVIEW

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('ssl smoke test', () => {
```
Comment (membphis) [inline web/cypress/integration/settings/settings-smoketest.spec.js:19]: "ssl smoke" ??

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    cy.get('#grafanaURL').clear().type('httx://www.test.com');
+    cy.get('.ant-form-item-explain').should('contain', 'Address is illegality');
+  });
+
+  it('set a accessible url', () => {
+    cy.visit('/');
+    cy.contains('Settings').click();
+    cy.wait(500);
+    cy.url().should('contains', '/settings');
+    cy.get('#grafanaURL').clear().type('https://www.baidu.com/');
```
Comment (membphis) [inline web/cypress/integration/settings/settings-smoketest.spec.js:57]: please change to another domain

**Review by membphis — CHANGES_REQUESTED** [review]

File: web/cypress/integration/settings/settings-smoketest.spec.js (hunk 1 above)
Comment (idbeta) [reply web/cypress/integration/settings/settings-smoketest.spec.js]: my fault, fixed.

File: web/cypress/integration/settings/settings-smoketest.spec.js (hunk 2 above)
Comment (idbeta) [reply web/cypress/integration/settings/settings-smoketest.spec.js]: fixed.

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 3]
```diff
@@ -16,7 +16,7 @@
  */
 /* eslint-disable no-undef */
 
-context('ssl smoke test', () => {
+context('settings psge smoke test', () => {
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js:19]: typo

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 4]
```diff
@@ -54,7 +54,7 @@ context('ssl smoke test', () => {
     cy.contains('Settings').click();
     cy.wait(500);
     cy.url().should('contains', '/settings');
-    cy.get('#grafanaURL').clear().type('https://www.baidu.com/');
+    cy.get('#grafanaURL').clear().type('https://www.apiseven.com/');
```
Comment (membphis) [inline web/cypress/integration/settings/settings-smoketest.spec.js:57]: no no no, that is a bad name.

we can use `github.com` or `apisix.apache.org`

**Review by membphis — CHANGES_REQUESTED** [review]

File: web/cypress/integration/settings/settings-smoketest.spec.js (hunk 3 above)
Comment (idbeta) [reply web/cypress/integration/settings/settings-smoketest.spec.js]: my fault, fixed.

File: web/cypress/integration/settings/settings-smoketest.spec.js (hunk 4 above)
Comment (idbeta) [reply web/cypress/integration/settings/settings-smoketest.spec.js]: good idea, fixed.

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('settings page smoke test', () => {
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js:19]:
```suggestion
context('Smoke Test for Setting page', () => {
```

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    pageContent: '.ant-pro-page-container',
+    notificationMsg: '.ant-notification-notice-message'
+  };
+
+  beforeEach(() => {
+    // init login 
+    cy.login();
+  })
+
+  it('visit settings page', () => {
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js:30]:
```suggestion
  it('should visit settings page', () => {
```

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    cy.wait(500);
+    cy.url().should('contains', '/settings');
+    cy.get(domSelectors.pageContent)
+      .children()
+      .should('contain', 'Setting')
+      .and('contain', 'Grafana Address')
+      .and('contain', 'Grafana address should begin with HTTP or HTTPS')
+  });
+
+  it('set a invaild url', () => {
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js:43]:
```suggestion
  it('should set a invaild url', () => {
```

**Review by juzhiyuan — APPROVED** [review]

**Comment by juzhiyuan** [comment]
ping @membphis @LiteSun

**Comment by juzhiyuan** [comment]
ping

File: web/cypress/integration/settings/settings-smoketest.spec.js (hunk 5 above)
Comment (membphis) [reply web/cypress/integration/settings/settings-smoketest.spec.js:19]: ping @idbeta

**Review by membphis — APPROVED** [review]

**Pull request merged**